### PR TITLE
NMS-13338: Add DnsDetector docs

### PR DESCRIPTION
**Pull request by mfuhrmann**
### All Contributors

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [x] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-13338

### CODE REVIEW

File: docs/modules/operation/pages/provisioning/detectors/DnsDetector.adoc
Comment (indigo423) [inline docs/modules/operation/pages/provisioning/detectors/DnsDetector.adoc:3]:
```suggestion
This detector lets you find DNS services for monitoring in your network.
```

File: docs/modules/operation/pages/provisioning/detectors/DnsDetector.adoc
Comment (indigo423) [inline docs/modules/operation/pages/provisioning/detectors/DnsDetector.adoc:4]:
```suggestion
The DNS service is detected when the server responds to a DNS query for an A resource record.
You can customize the A record used in the query with the `lookup` configuration parameter.
```

**Review by indigo423 — COMMENTED** [review]
Minor changes

File: docs/modules/operation/pages/provisioning/detectors/DnsDetector.adoc
Comment (Bonrob2) [inline docs/modules/operation/pages/provisioning/detectors/DnsDetector.adoc:3]:
```suggestion
This detector lets you find DNS services to monitor in your network.
```
Please verify this update is correct.

File: docs/modules/operation/pages/provisioning/detectors/DnsDetector.adoc
Comment (Bonrob2) [inline docs/modules/operation/pages/provisioning/detectors/DnsDetector.adoc:6]:
```suggestion
Use the command line tool `host` to simulate the detector:
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]

**Comment by mfuhrmann** [comment]
@Bonrob2 @indigo423 It is more or less the text from the monitors. I thought I replaced the "monitoring" part against "detection". But looks like I forgot some. Thanks!

**Comment by Bonrob2** [comment]
> @Bonrob2 @indigo423 It is more or less the text from the monitors. I thought I replaced the "monitoring" part against "detection". But looks like I forgot some. Thanks!

> @Bonrob2 @indigo423 It is more or less the text from the monitors. I thought I replaced the "monitoring" part against "detection". But looks like I forgot some. Thanks!

Hi @mfuhrmann, I still see my previous suggestions for update, but otherwise it looks fine.

**Review by Bonrob2 — APPROVED** [review]

**Review by indigo423 — APPROVED** [review]
LGTM!

**Comment by indigo423** [comment]
@mfuhrmann you can go ahead and merge them 2 green approved reviewers is good enough :) After merge cleanup: Update the JIRA issue and delete the branch.

**Comment by indigo423** [comment]
@mfuhrmann you can go ahead and merge them 2 green approved reviewers is good enough :) After merge cleanup: Update the JIRA issue and delete the branch.

**Pull request merged**
